app/containers/Facilities/FacilitiesRowItemContainer.tsx

```diff
@@ -26,6 +26,15 @@ export const FacilitiesRowItemComponent: React.FunctionComponent<Props> = ({
   query
 }) => {
   const {applicationRevisionStatus} = facilitySearchResult;
+  const isPendingCertification = Boolean(
```

Comment:
would this be better renamed to `hasCertificationUrl`, since an application is only pending certification when it has a url *and* is a draft